docs/source/yelpsoa_configs.rst

```diff
@@ -51,6 +51,12 @@ instance MAY have:
     namespace. For example ``canary`` instances can have ``nerve_ns: main`` to route
     their traffic to the same pool as the other ``main`` instances.
 
+  .. _net:
+
+  * ``net``: Specify which kind of
```

Comment:
please add a link to reference this in chronos as well. I know we don't currently use this much in chronos, but someone will probably want networking in chronos eventually